sdk/identity/tests.yml

```diff
@@ -3,6 +3,21 @@ trigger: none
 extends:
   template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
   parameters:
+    AdditionalPlatforms:
+      displayName: 'Linux with keyring'
+      vmImage: 'ubuntu-18.04'
+      container:
+        image: runtestsincontainer.azurecr.io/experiment/run_tests_in_docker:latest
```

Comment:
This is a temporary Docker image. We either to decide on where the images should be.

Reply:
Yes we need to figure out the correct place to host the images. We should probably setup a container registry in our EngSys subscription. @danieljurek @mitchdenny can one of you guys please set that up?

For now we should just commit the DockerFile for this container into the repo and we can manually create and push it. Once we get more containers to maintain we can setup some automated pipelines to build and publish updates.

Reply:
My experience with Docker is rather limited but I recall that Docker works by default with files named `Dockerfile`. To that end I think we should put the Dockerfile in something like `eng/containers/<container-name>/Dockerfile` ... What do people think?

Reply:
That naming convention seems reasonable to me.

Reply:
I just posted more about the container registry I setup in our teams channel.